dataprofiler/profilers/profiler_options.py

```diff
@@ -1274,6 +1281,25 @@ def _validate_helper(self, variable_path: str = "StructuredOptions") -> list[str
                 "a re.RegexFlag".format(variable_path)
             )
 
+        if self.sampling_ratio is not None and not isinstance(
+            self.sampling_ratio, float
+        ):
+            errors.append(
+                "{}.sampling_ratio must be either None or an float".format(
```

Comment:
small fix: `or a float`

Reply:
+1

Reply:
Would it be worth to accept an `int` bc `1` is valid.

Reply:
yeah I was thinking that too actually -- I think that's a fair point